macros/plugins/bigquery/create_external_table.sql

```diff
@@ -45,5 +45,8 @@
                 {%- endif -%}
             {%- endfor -%}
             {%- endif -%}
+            {%- if hours_to_expiration -%}
+                , expiration_timestamp = TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL {{hours_to_expiration}} hour)
```

Comment:
can we integrate this into the already existing enumeration of `options.items()` (`L39-L47`)? perhaps something like the below?

this solution isn't very clean either though, imho. I thought to use the fancy jinja `loop.first` to exclude the leading comma in the first example, but `expiration_timestamp` will never be first in the list of options as `uris` are a required option. 😵 . perhaps we could refactor so that the currently unused `external` field is where `uris` are given??

```
            {%- if options is mapping -%}
            {%- for key, value in options.items() if key != 'uris' %}
                {%- if value is string -%}
                    , {{key}} = '{{value}}'
                {%- else -%}
                    {%- if key == "hours_to_expiration" -%}
                        , expiration_timestamp = TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL {{hours_to_expiration}} hour)
                    {%- else -%}
                        , {{key}} = {{value}}
                {%- endif -%}
            {%- endfor -%}
            {%- endif -%}
```